src/main/webapp/dev/js/common/feedbackResponseComments.es6

```diff
@@ -144,6 +150,141 @@ function showErrorMessage(errorMessage, submitButton) {
     submitButton.html('<span class="glyphicon glyphicon-trash glyphicon-primary"></span>');
 }
 
+function enableHoverToDisplayEditOptions() {
+    // show on hover for comment
+    $('body').on('mouseenter', '.comments > .list-group-item', function () {
+        $('div[id|="commentBar"] a[type="button"]', this).show();
+    });
+
+    $('body').on('mouseleave', '.comments > .list-group-item', function () {
+        $('div[id|="commentBar"] a[type="button"]', this).hide();
+    });
+}
+
+function showResponseCommentAddForm(recipientIndex, giverIndex, qnIndex, sectionIndex) {
+    const id = `${sectionIndex !== undefined ? `-${sectionIndex}` : ''}-${recipientIndex}-${giverIndex}-${qnIndex}`;
+
+    $(`#responseCommentTable${id}`).show();
+    if ($(`#responseCommentTable${id} > li`).length <= 1) {
+        $(`#responseCommentTable${id}`).css('margin-top', '15px');
+    }
+    $(`#showResponseCommentAddForm${id}`).show();
+
+    $(`#responseCommentAddForm${id}`).empty();
+
+    if (typeof richTextEditorBuilder !== 'undefined') {
+        /* eslint-disable camelcase */ // The property names are determined by external library (tinymce)
+        richTextEditorBuilder.initEditor(`#responseCommentAddForm${id}`, {
+            inline: true,
+        });
+        /* eslint-enable camelcase */
+    }
+    saveInitialVisibilityOfCheckboxes(`showResponseCommentAddForm${id}`,
+        $(`#showResponseCommentAddForm${id}`).children('.responseCommentAddForm'));
+    $(`#responseCommentAddForm${id}`).focus();
+}
+
+function hideResponseCommentAddForm(recipientIndex, giverIndex, qnIndex, sectionIndex) {
+    const id = `${sectionIndex !== undefined ? `-${sectionIndex}` : ''}-${recipientIndex}-${giverIndex}-${qnIndex}`;
+
+    if ($(`#responseCommentTable${id} > li`).length <= 1) {
+        $(`#responseCommentTable${id}`).css('margin-top', '0');
+        $(`#responseCommentTable${id}`).hide();
+    }
+    restoreInitialVisibilityOfCheckboxes(
+        $(`#showResponseCommentAddForm${id} > form`), $(`#showResponseCommentAddForm${id}`));
+    $(`#showResponseCommentAddForm${id}`).hide();
+    removeFormErrorMessage($(`#button_save_comment_for_add${id}`));
+}
+
+function showResponseCommentEditForm(recipientIndex, giverIndex, qnIndex, commentIndex, sectionIndex, viewType) {
+    let id;
+
+    if (`${sectionIndex}` !== 'undefined') {
+        id = `-${sectionIndex}-${recipientIndex}-${giverIndex}-${qnIndex}-${commentIndex}`;
+    } else if (`${viewType}` !== 'undefined') {
+        id = `-${viewType}-${recipientIndex}-${giverIndex}-${qnIndex}-${commentIndex}`;
+    } else {
+        id = `-${recipientIndex}-${giverIndex}-${qnIndex}-${commentIndex}`;
+    }
+
+    const commentBar = $(`#plainCommentText${id}`).parent().find(`#commentBar${id}`);
```

Comment:
Prefix this variable name with `$` to indicate it is a jQuery object.

Reply:
This code is not written by me. I have just moved code to a different position to avoid lint errors. Should I change it?

Reply:
I'd suggest changing these while you're moving it (since git already flags the line as a change anyway), and by "move" I suppose the old code is no longer there (ie. this is not a "copy" action). As long as it doesn't alter the behaviour of the code it should be fine.